tiledb/bioimg/converters/ome_zarr.py

```diff
@@ -79,6 +80,50 @@ def group_metadata(self) -> Dict[str, Any]:
         )
         return {"json_zarrwriter_kwargs": json.dumps(writer_kwargs)}
 
+    @property
+    def image_metadata(self) -> Dict[str, Any]:
+        metadata: Dict[str, Any] = {}
+        node_metadata = self._multiscales.node.metadata
+        color_generator = iter_color(np.dtype(np.uint8))
+        channels = []
+        for idx in range(len(node_metadata.get("name"))):
```

Comment:
Add some comments since most of this code is `format` related and not maintainable.  If the `self._multiscales.node.metadata` structure changes this will need full refactor.